environments/docker.toml

```diff
@@ -165,6 +159,12 @@
         migrations = "db/migrations"
         contracts = ["MCD_CAT"]
         rank = "0"
+    [exporter.cdp_manager]
```

Comment:
Why move this? I kinda like having the storage transformers alphabetical and then the event transformers alphabetical, though would also be open to strict alphabetical

Reply:
Ah.. I don't think I noticed that order. I had deleted that to test a failing case in travis and put it back out of order.